cli/src/factory_ops/unboxing/init

```diff
@@ -303,7 +303,7 @@ sleep 2
 echo "Bringing HCTL cluster out of maintenance mode" 2>&1 | tee -a ${LOG_FILE}
 hctl node unmaintenance --all 2>&1 | tee -a ${LOG_FILE}
 
-echo "Ensure cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
+echo "Ensuring the cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
```

Comment:
Did we not changed this message - "Restarting the cluster resources through switching maintenance mode"